### ISSUE TITLE
Is is possible to send the UDP packet with incremental sequence number

### ISSUE THREAD

**Issue by ricaonline** [dannagle/PacketSender#110]
#### What OS?

- [x ] Windows
- [ ] Mac
- [ ] Linux (Which distro?)

#### Description of issue

In my test scenario, I need to send the UDP packet with incremental sequence number like in the beginning of the UDP data(HEX content), in order to compare on the receiver side the number and sent/received time for the same packet. Packet Sender seems only send the same packet at every intervals, is there anyway to set incremental sequence number like macro or any script to do it?

**Comment by dannagle** [comment]
Correct. Packet Sender's macro system only works during responses. It does not have macro support for initial sending. I agree that would be nice to have. 

I offer feature bounty services if you'd like macros for sending added. You could potentially reuse the {{TIME}} or {{UNIQUE}} macros or perhaps I could make a brand new macro {{COUNTER}} if that works best for you. 

Is this something you'd be interested in sponsoring? If so, you could email me directly. My rates are very reasonable for feature requests that can potentially help everybody (like this one).  https://packetsender.com/contact

**Comment by andy778** [comment]
This would have been use-full for me also. It should be just to add in 
https://github.com/dannagle/PacketSender/blob/master/src/packet.cpp#L846
and a variable that $Counter++

**Comment by dannagle** [comment]
This is an old ticket that is now out of date. Macros are now supported during sending.  They are also a lot easier for me to add now than when this ticket was first filed.

You got the right area to add a `{{COUNTER}}`, but instead of a global, I think a static unsigned quint32 within that function would work.  I will add this to my list for next release. 

If you need something immediately, there is now {{UNIXTIME}} that will get you a an incrementing number at a rate of 1 Hz. If you need something faster, you can also customize {{DATE}} and {{TIME}} to get milliseconds.

**Comment by dannagle** [comment]
This is now in the development branch and will be in the next release.
